docs/custom_model_loader.md

```diff
@@ -0,0 +1,60 @@
+Follow the below steps to use example custom model loader:
+
+Step 1: Prepare Docker*
+
+Download the model server code & build the docker (make docker_build).
+Once the docker is ready, create a folder where all the artefacts can be downloaded. Ensure the models, client components, images are all downloaded to this folder. Also create the json required into this folder.
+% mkdir test_custom_loader
+
+
+Step 2: Download a Model
+
+% curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.4/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.4/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/fdsample/1/face-detection-retail-0004.xml -o model/fdsample/1/face-detection-retail-0004.bin
```

Comment:
commands should be in ``` <command>``` formatting to be well visible in github.

Reply:
Changed in the document